docker/monero/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Download Haveno's custom Monero binaries. Managed by docker-compose.
+
+FROM ubuntu:20.04 as downloader
+
+RUN set -ex && \
+    apt update && \
+    apt --no-install-recommends --yes install \
+        wget \
+        ca-certificates
+
+# We use the existent haveno_deps.sh script to download and verify the Monero binaries
+RUN set -ex && wget https://raw.githubusercontent.com/napoly/haveno/docker/scripts/haveno_deps.sh && \
```

Comment:
should be changed to https://raw.githubusercontent.com/haveno-dex/haveno/master/scripts/haveno_deps.sh before merging to master